Wbooks/BookRepository.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BookRepository.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 19/04/2021.
+//
+
+import Alamofire
+
+internal class BookRepository {
```

Comment:
`internal` is the default access level, so you can remove the keyword if you like. 
Apart from that, this could a struct, which would be even better 👀